src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -107,6 +111,52 @@ function renderComponent(postId) {
   return wrapper;
 }
 
+describe('PostView', () => {
+  const threadId = 'thread-1';
+  beforeEach(() => {
+    initializeMockApp({
+      authenticatedUser: {
+        userId: 3,
+        username: 'abc123',
+        administrator: true,
+        roles: [],
+      },
+    });
+
+    store = initializeStore();
+    Factory.resetAll();
+    axiosMock = new MockAdapter(getAuthenticatedHttpClient());
+    axiosMock.onGet(`${threadsApiUrl}${threadId}/`)
+      .reply(200, Factory.build('thread'));
```

Comment:
generate thread data corresponding to the id of the required topic would be the batter approach.